DataGrid Getting Started tutorial: change 'Export Data' section (#3677)
DevExpress/devextreme-documentation: concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data.md

> DataGrid allows users to export grid data as Excel and PDF documents. This feature requires the following third-party libraries:

Reviewer: ```suggestion
The DataGrid allows users to export grid data as Excel and PDF documents. This feature requires the following third-party libraries:
```

- Excel: <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v4+ and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> v2.0.2+

- PDF: <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a>

> To configure export operations, use the **excelExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or **pdfExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method. It requires a DataGrid instance and a target Excel worksheet or a PDF document.

Reviewer 2: You have this paragraph about `exportDataGrid` methods, but then the first sentence in the next paragraphs continues to talk about it. It even refers to it as "this method". I would rather have it all in one paragraph and then you can start the next paragraph with a sentence about the example. 

```suggestion
To configure export operations, use the **excelExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or **pdfExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method. It requires a DataGrid instance and a target Excel worksheet or a PDF document. You can call these methods at any point in your application.
```

> The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available via the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats]() property to configure available export operation types.

Author: ```suggestion
The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available via the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#formats) property to configure available export operation types.
```

> You can call this methods at any point in your application. The following code calls them in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler is executed when users click the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats]() property to configure which types of export the button shows.

Reviewer 2: The first sentence moved to the previous paragraph. 

Formats property seems to be missing a link. 

I also made a couple of changes to get rid of passive, repetition (onExporting mentioned in two consecutive sentences), and a "which" clause in the last sentence. 

```suggestion
The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available via the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats]() property to configure available export operation types.
```

> The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available via the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#formats) property to configure available export operation types.

Reviewer: ```suggestion
The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available using the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#formats) property to configure available export operation types.
```

Author: I decided to rephrase this sentence since I don't want to use 'using' in the sentences.
What do you think about the following?

> The control executes this handler when users select one of the operations available in the **Export** button drop-down menu.

Reviewer: this works

    import { jsPDF } from 'jspdf';
    import { exportDataGrid as exportDataGridToPdf} from 'devextreme/pdf_exporter';
    // ...

    const exportFormats = ['xlsx', 'pdf'];

    function exportGrid(e) {
        if (e.format === 'xlsx') {
            const workbook = new Workbook(); 
            const worksheet = workbook.addWorksheet("Main sheet"); 
            exportDataGrid({ 
                worksheet: worksheet, 
                component: e.component,
            }).then(function() {
                workbook.xlsx.writeBuffer().then(function(buffer) { 
                    saveAs(new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx"); 
                }); 
            }); 
            e.cancel = true;
        } 
        else if (e.format === 'pdf') {
            const doc = new jsPDF();
            exportDataGridToPdf({
                jsPDFDocument: doc,
                component: e.component,
            }).then(() => {
                doc.save('DataGrid.pdf');
            });
        }
    }

    function App() {
        // ...
        return (
            <DataGrid ...
                onExporting={exportGrid}>
                {/* ... */}

>                 <Export enabled={true} formats={exportFormats}>

Author: ```suggestion
                <Export enabled={true} formats={exportFormats} />
```
